plugins/outputs/redistimeseries/redistimeseries.go

```diff
@@ -0,0 +1,88 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  # Redis ACL username
+  # username = ""
+  ## password to login Redis
+  # password = ""
+  database = 0
+  ## Optional TLS Config
```

Comment:
Please add a space between the two config option blocks and maybe move the `database` option up closer to `address`? Anyway, please document the `database` option and comment it as zero is the default anyway.

Reply:
I'm working on these now.